docs/admin.md

```diff
@@ -426,6 +426,35 @@ Content-Type: application/zip
 Content-Disposition: attachment; filename="alice.cozy.localhost - part001.zip"
 ```
 
+## OAuth clients usage
+
+### GET /instances/:domain/clients-usage
```

Comment:
I don't understand why we have two endpoints that do the same thing.

Reply:
This route is an admin one. I used the same pattern as the disk usage one.

Reply:
hmmm, I don't see why we have two routes for disk usage too...

Reply:
To have different permissions checks? Or to avoid handling JSON-API data in the command line?

Reply:
Maybe, but it looks like bad reasons for both cases.

Reply:
OK. Since this admin route for clients usage is not used yet, I can remove it. We can figure out what to do with the disk usage one later.